### ISSUE TITLE
Add ability for channel/thread links to be elements

### ISSUE THREAD

**Issue by kylepokorski** [rayzchen/pyeod#72]
It allows the creation of more interesting elements.

**Comment by srimpsrimp** [comment]
this also applies to emojis, why can't emojis be put in elements?

**Comment by Cheesy-Brik** [comment]
Discord has internal custom unicode formatting for emotes and discord links. This means that you can't even copy and paste as you have to use the actual discord character. For emotes this means you must have discord nitro for off server emotes, and for channels, if a private channel is referenced and you do not have access to it, a seperate discord character will be outputed meaning you cannot access the element. While technically you could access these elements by id this would get annoying fast and it's not worth dealing with all the baggage discord's fucked up characters come with.
Tldr: Problem with discord, not us.
